src/deva/config/model/git.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2024-present Datadog, Inc. <dev@datadoghq.com>
+#
+# SPDX-License-Identifier: MIT
+from __future__ import annotations
+
+from msgspec import Struct, field
+
+
+def _get_config(env_var: str, config_name: str) -> str:
+    import os
+
+    value = os.environ.get(env_var, "")
+    if value:
+        return value
+
+    import subprocess
+
+    try:
+        return subprocess.run(
+            ["git", "config", "--global", "--get", config_name],  # noqa: S607
+            encoding="utf-8",
+            capture_output=True,
+            check=True,
+        ).stdout.strip()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
With invoke we avoided using `subprocess` and rather use the [run](https://docs.pyinvoke.org/en/stable/api/runners.html#invoke.runners.Runner.run) function in its context. Would it be possible to do the same with Click?